packages/quick_actions/quick_actions/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
     sdk: flutter
   quick_actions_android: ^1.0.0
   quick_actions_ios: ^1.0.0
```

Comment:
The iOS part needs to land in a separate PR and this constraint updated to require it, otherwise someone who updates by editing their pubspec.yaml instead of running `pub update` will get a set of versions that claims to support `localizedTitle`, allows them to pass `localizedTitle`, but ignores that parameter. See [step 5 of the instructions for federated plugins](https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changing-federated-plugins).

Reply:
Thank you for the feedback. I have created this [PR]（ https://github.com/flutter/packages/pull/8149 ).